luigi/execution_summary.py

```diff
@@ -276,6 +278,7 @@ def _get_comments(group_tasks):
 _COMMENTS = set((
     ("already_done", 'present dependencies were encountered'),
     ("completed", 'ran successfully'),
+    ("ever_failed", 'ever failed'),
```

Comment:
Actually, I think it's better to not print this one out. I don't think it's so relevant. We should keep the execution *summary* short. What do you think?